kafkajobs/driver.go

```diff
@@ -236,6 +236,18 @@ func (d *Driver) Run(ctx context.Context, p jobs.Pipeline) error {
 		return errors.E(op, errors.Errorf("no such pipeline registered: %s", pipe.Name()))
 	}
 
+	if d.cfg.Ping != nil {
```

Comment:
It's better to use `ping` inside the `FromConfig`/`FromPipeline` methods. They're called first to initialize the Kafka pipeline.

Reply:
I also did this at first, but I thought that there might be problems with bootstrap.

Reply:
RR will surely return an error. It is better to return an error as soon as possible to stop the boot and inform the user.